### PR TITLE
Feat/2958 table content

### PR DESCRIPTION
**Pull request by adipaul1981**

### CODE REVIEW

File: packages/ui/src/components/tables/TableContent.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+export type TableContentProps = {
+    className?: string;
+    children?: React.ReactNode;
+};
+
+const TableContent: React.FC<TableContentProps> = ({
+    children,
+    className = '',
+}: TableContentProps) => {
+    return (
```
Comment (francisl) [inline packages/ui/src/components/tables/TableContent.tsx:14]: no return needed here

File: packages/ui/src/components/tables/TableContent.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+export type TableContentProps = {
+    className?: string;
+    children?: React.ReactNode;
+};
+
+const TableContent: React.FC<TableContentProps> = ({
+    children,
+    className = '',
+}: TableContentProps) => {
```
Comment (francisl) [inline packages/ui/src/components/tables/TableContent.tsx:13]: no need to type props

File: storybook/stories/Components/Tables/TableContent.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+const TableContentStory = ({title, style, ...props} : {title: string, style: React.CSSProperties, props: Story<TableContentProps>}) => (
+  <>
+    <h3>{title}</h3>
+    <TableContent {...props}>{props.children}</TableContent>
+  </>
+);
+
+export const StackLayoutHorizontal = TableContentStory.bind({});
+const arrayLength = Array.from(Array(20).keys())
+StackLayoutHorizontal.args = {
```
Comment (francisl) [inline storybook/stories/Components/Tables/TableContent.stories.tsx:30]: ?

File: storybook/stories/Components/Tables/TableContent.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+} as Meta;
+
+const TableContentStory = ({title, style, ...props} : {title: string, style: React.CSSProperties, props: Story<TableContentProps>}) => (
+  <>
+    <h3>{title}</h3>
+    <TableContent {...props}>{props.children}</TableContent>
+  </>
+);
+
+export const StackLayoutHorizontal = TableContentStory.bind({});
```
Comment (francisl) [inline storybook/stories/Components/Tables/TableContent.stories.tsx:28]: the name will be the shown in the sidebar. It need to be descriptive

**Review by francisl — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/tables/TableContent.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import '@ferlab/style/components/tables/TableContent.scss'
+
+export type TableContentProps = {
+    className?: string;
+    children?: React.ReactNode;
+};
+
+const TableContent: React.FC<TableContentProps> = ({
```
Comment (kjacqueschu) [inline packages/ui/src/components/tables/TableContent.tsx:10]: Not sure about the name of the component though... ScrollableTable was saying more on what was the component doing

File: storybook/stories/Components/Tables/TableContent.stories.tsx (hunk 3 above)
Comment (adipaul1981) [reply storybook/stories/Components/Tables/TableContent.stories.tsx:30]: wanted a table with 20 rows...

**Pull request merged**